packages/babel-node/test/fixtures.js

```diff
@@ -102,7 +102,13 @@ const buildTest = function (testName, opts) {
     saveInFiles(opts.inFiles);
     const args = [binLoc].concat(opts.args);
 
-    const spawnOpts = { cwd: tmpLoc, env: { BABEL_DISABLE_CACHE: true } };
+    const spawnOpts = {
+      cwd: tmpLoc,
+      env: {
+        ...(process.env.BABEL_8_BREAKING && { BABEL_8_BREAKING: true }),
```

Comment:
Can we pass through the `process.env` plus `BABEL_DISABLE_CACHE: true`? Are there any environment variables that we don't want to pass through?

Reply:
Some of the flags in `process.env` indicate that the terminal supports colors, and passing them to the spawned process means that, in terminals with those env flags, the generated fixtures have the ANSII escape codes.